assayist/processor/post_analyzer.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import glob
+import os
+
+from hashlib import sha256
+
+from assayist.common.models import content
+from assayist.processor.base import Analyzer
+from assayist.processor.logging import log
+
+BLOCKSIZE = 65536
+
+
+class PostAnalyzer(Analyzer):
+    """Performs post-analysis."""
+
+    def run(self):
+        """Start the post analyzer."""
+        build_info = self.read_metadata_file(self.BUILD_FILE)
+        build_id = build_info['id']
+
+        if not self.is_container_build(build_info):
+            # Post analysis consists of recording unknown files, which only makes sense for
+            # container builds. RPM or maven builds will not include any unindentified files.
+            log.info(f'Skipping build {build_id} because the build is not a container')
+            return
+
+        # Remember current directory
+        orig_dir = os.getcwd()
+
+        # Switch to dir of all unpacked content to search for RPM files
+        unpacked_container_layer = os.path.join(self.input_dir, self.UNPACKED_CONTAINER_LAYER_DIR)
+        os.chdir(unpacked_container_layer)
+
+        for archive in os.listdir():
+            for unknown_file in glob.iglob('**', recursive=True):
+                if os.path.isdir(unknown_file):
+                    continue
+
+                # Assume that the artifact being analyzed was created by the main analyzer.
+                archive_obj = content.Artifact.nodes.get(filename=archive)
+
+                path_to_archive = os.path.join(unpacked_container_layer, archive)
+                path, filename = os.path.split(os.path.relpath(unknown_file, path_to_archive))
+
+                unknown_file = content.UnknownFile.get_or_create({
+                    'checksum': self.sha256_checksum(unknown_file),
+                    'filename': filename,
+                    'path': '/' + path,  # Add leading root dir
```

Comment:
Why do we add a leading slash? It would be nice if a comment in assayist/common/models/content.py explicitly said whether this is expected.

Reply:
I guess I would expect it, otherwise it looks like a relative path but it's not clear what it is relative to. I can certainly add a comment to note that the path is an absolute path.

Reply:
Added!